mllib/src/test/scala/org/apache/spark/ml/feature/QuantileDiscretizerSuite.scala

```diff
@@ -52,6 +52,25 @@ class QuantileDiscretizerSuite
       "Bucket sizes are not within expected relative error tolerance.")
   }
 
+  test("Test Bucketizer on duplicated splits") {
+    val spark = this.spark
```

Comment:
Hm, I'm not sure why the tests write this. It shouldn't be needed. Leave it for now I guess